hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/ExponentialRetryPolicy.java

```diff
@@ -75,23 +57,16 @@ public class ExponentialRetryPolicy extends AbfsRetryPolicy {
    */
   private final int minBackoff;
 
-  /**
-   * Initializes a new instance of the {@link ExponentialRetryPolicy} class.
-   */
-  public ExponentialRetryPolicy(final int maxIoRetries) {
```

Comment:
I would suggest rather than changing the below constructor, add a param to this constructor to accept abfsCongifuration and use the backoff values from abfsConfiguration. This would prevent null passing across.

Reply:
Makes sense. Taken

Reply:
I think this is not what Anmol suggested.
I think she meant to keep the below constructor untouched and update this contructor to accept absfConfiguation

Reply:
That is correct, we should not change the default constructor.